apps/daimo-mobile/src/view/shared/SwipeUpDown.tsx

```diff
@@ -221,3 +259,10 @@ const styles = StyleSheet.create({
     backgroundColor: color.white,
   },
 });
+
+const ANIMATION_CONFIG = {
```

Comment:
hmm is my understanding correct that this is what changes the bounciness (and the rest of the PR is the change for transitions of content inside in the bottom sheet?). At that point, our bottom sheet would feel as playful to move as Apple's but the transitions inside would be as-before?

If so, for code quality / maintainability I would recommend just making the bounciness change and leave the in-sheet content as is. Apologies for the extra work, didn't realize it would be require such a deep change.

Reply:
Unfortunately if we change config inside our components it triggers rerender of bottom sheet and calls some useEffects inside of the library and as a result the bottom sheet tries to close after config change and it's a constant struggle with auto closing modal. 

I tried to turn off some useEffects that trigger auto close inside the library but even thou it looked like it worked, I am afraid that it wont trigger in some edge cases we want it to. 

This way or another some changes inside the library are necessary to achieve multiple animation configs 😞